Up2dateService/Up2dateShared/InstallPackageStatus.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Up2dateShared
+{
+    public enum InstallPackageStatus
+    {
+        Ok,
+        PackageUnavailable,
+        TempDirectoryFail,
+        DataCannotBeRead,
```

Comment:
Bad name. Be more precise.
Maybe InvalidChocoPackage?

Reply:
Replaced